Gemfile

```diff
@@ -1,3 +1,4 @@
-gem "rake"
+source 'https://rubygems.org'
+gem "rake", '<11.0'
 gem "semver", "~> 1.0.1"
-gem "albacore"
\ No newline at end of file
+gem "albacore", "0.3.6"
```

Comment:
I think it is time to remove ruby from the build process. With the .NET SDK tooling performing most of the work we've been able to do the few additional things as part of that process with Castle Core and Windsor.

Reply:
I think he wanted to separate PRs; this PR does remove the build's dependency on it. Another can remove it.